client/src/components/FundCard.tsx

```diff
@@ -23,12 +24,18 @@ const FundCard = ({
   image,
   handleClick,
 }: CardType) => {
+  const { address } = useStateContext();
+  const isOwner = owner === address;
+
   const remainingDays = daysLeft(deadline);
   return (
     <div
-      className="sm:w-[288px] w-full rounded-[15px] bg-[#1c1c24] cursor-pointer"
+      className="sm:w-[288px] w-full rounded-[15px] bg-[#1c1c24] cursor-pointer relative"
       onClick={handleClick}
     >
+
+      {isOwner && <p className="z-[100] absolute right-[15px] top-[10px] text-[red]">Edit</p>}
```

Comment:
You use **isOwner** only one time. It's not necessary to create a new variable in one-usage case. 

```suggestion
      {owner === address && <p className="z-[100] absolute right-[15px] top-[10px] text-[red]">Edit</p>}
```